packages/mobile/src/paymentRequest/PaymentRequestConfirmation.tsx

```diff
@@ -112,22 +113,14 @@ class RequestConfirmation extends React.Component<Props> {
       notified: false,
     }
 
-    if (requesteeAddress) {
-      try {
-        this.props.writePaymentRequest(paymentInfo)
-      } catch (error) {
-        Logger.error(TAG, 'Payment request failed, show error message', error)
-        this.props.showError(ErrorMessages.PAYMENT_REQUEST_FAILED)
-        return
-      }
-    } else {
-      // TODO: handle unverified recepients, maybe send them a sms to download the app?
-      this.props.showError(ErrorMessages.CAN_NOT_REQUEST_FROM_UNVERIFIED)
-      Logger.info(
-        'RequestConfirmation/onConfirm',
-        'Currently requesting from unverified users is not supported'
-      )
```

Comment:
I assume we do support payment requests using only an address now?

Reply:
Correct, this has always been the case, this file was stale